scripts/node_e2e_installer

```diff
@@ -30,6 +31,13 @@ install_crio() {
     # Setup log level
     echo "CONTAINER_LOG_LEVEL=debug" >>/etc/sysconfig/crio
 
+    for condition in "true" "True" "TRUE"; do
+        if [[ "${ENABLE_POD_EVENTS}" = "$condition" ]]; then
+            echo "ENABLE_POD_EVENTS=true" >>/etc/sysconfig/crio
```

Comment:
this likely is a follow up because I see some other examples, but I think we should use drop-in files instead of env variables in sysconfig files, as I would say it's a preferred method of configuring cri-o

Reply:
Okay